### DISCUSSION TITLE
How to prevent automatic generation of <link rel="alternate"> and <meta property="og:locale:alternate"> tags?

### DISCUSSION THREAD

**Discussion by leopoldkristjansson** [nuxt-modules/i18n#2786] [Q&A]
I want to clarify whether I am doing something wrong here - or possibly this is a bug?

This is a simplified version of my `/layouts/default.vue` file.

```vue
<script setup lang="ts">
const head = useLocaleHead({
  addDirAttribute: true,
  identifierAttribute: 'id',
  addSeoAttributes: true,
})

const shouldHaveNoIndexMarker = isStaging
</script>

<template>
  <Html
    :lang="head.htmlAttrs?.lang"
    :dir="head.htmlAttrs?.dir"
  >
    <Head>
      <Meta
        v-if="shouldHaveNoIndexMarker"
        name="robots"
        content="noindex, nofollow, noarchive, nosnippet"
      />
      <template v-for="link in head.link" :key="link.id">
        <Link :id="link.id" :rel="link.rel" :href="link.href" :hreflang="link.hreflang" />
      </template>
      <template v-for="meta in head.meta" :key="meta.id">
        <Meta :id="meta.id" :property="meta.property" :content="meta.content" />
      </template>
    </Head>
    <Body>
      <slot />
    </Body>
  </Html>
</template>
```

So, the issues I am having are as follows:

- If I set `addSeoAttributes` to `false` and I remove the v-for loops inside of `<Head>` I still get all the `<link rel="alternate">` and `<meta property="og:locale:alternate">` tags generated in my `<head>` tag? It looks to me as the v-for loops are not in control of anything.
- TS is not working as I would expect. For example, setting `addSeoAttributes: 'foobar',` does not make VS Code complain.

In short, I am looking for more control over the metadata in the `<head>` and I am not seeing exactly how to go about it.

Thanks in advance!
